webmin/letsencrypt-cleanup.pl

```diff
@@ -67,4 +67,11 @@
 	&virtual_server::reload_bind_records($d);
 	}
 
+# Clean up any dynamic record
+use Net::DNS;
```

Comment:
What if the Net::DNS perl module isn't installed?

Reply:
Valid point - I'd assumed it was a dependency of webmin already as it was installed on my system, but having looked it's actually a dependency of virtualmin.

In which case I'm not sure yet what the right approach to this is. `nsupdate` could be used, but the same question goes for that. It might work by using webmin APIs to freeze, modify and thaw the zone. I'll have to do some research and experimentation.

Reply:
I'll merge this, and add some code to support the case where those deps are missing.

Reply:
Actually ... I think the freeze/thaw approach is better, because that's what Webmin normally does.

Reply:
Check out this patch : https://github.com/webmin/webmin/commit/94c64c2308328118de5a1cff45165413f82edab3

Reply:
Amazing, just tested and that works by the look of it! Thank you! 👍